__tests__/app.test.js

```diff
@@ -329,3 +327,22 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  it("200: responds with array of all users with correct properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const users = body.users;
```

Comment:
aim for consistency with destructruing v dot notation. As you have destructred the function parameter you should also destructure here:

`const {users} = body`

If you want to be really fancy you can do a bit of nested destructuring:

`({ body : { users } } )=>{}`